### PR TITLE
[FIX] path import

### PR DESCRIPTION
**Pull request by rousseldenis**
@sbidoul

### CODE REVIEW

**Comment by rousseldenis** [comment]
image

**Review by sbidoul — APPROVED** [review]
Tested, works file.

Cherry on the cake would be setting the lowest pytest version that supports this path argument.

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     include_package_data=True,
     platforms='any',
     install_requires=[
-        'pytest>=2.9',
+        'pytest>=7.0.0',
```
Comment (rousseldenis) [inline setup.py:21]: @sbidoul Done. Actually, the problem appears with the lastest version

**Review by mmequignon — APPROVED** [review]
Thanks !

**Comment by sbidoul** [comment]
Note that pathlib is python 3 only, and pytest 7 is python 3.6+. So this patch effectively means abandoning python 2 and there should be a `requires-python = ">=3.6"` in setup.py to declare that cleanly.

**Comment by simahawk** [comment]
> Note that pathlib is python 3 only, and pytest 7 is python 3.6+. So this patch effectively means abandoning python 2 and there should be a `requires-python = ">=3.6"` in setup.py to declare that cleanly.

Agree. If you want to use py2 then you can pin <= 0.6.1.

**Comment by sbidoul** [comment]
> If you want to use py2 then you can pin <= 0.6.1.

Yes, but please add `python_requires = ">=3.6"` in setup.py, so the next release will not be picked up by pip on py2.

**Comment by rousseldenis** [comment]
> > If you want to use py2 then you can pin <= 0.6.1.
> 
> Yes, but please add `requires_python = ">=3.6"` in setup.py, so the next release will not be picked up by pip on py2.

Done

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
there you go https://pypi.org/project/pytest-odoo/0.7.1/

I had to add a patch because I realized the README markup was broken only after tagging :sweat:

**Pull request merged**